### DISCUSSION TITLE
What other alternatives are there to Heroku?

### DISCUSSION THREAD

**Discussion by developer-luis** [PracticalBooks/Practical-Laravel#4] [General]
I have followed the book to the letter but when I get to chapter 30 (Deploying to the Cloud - Heroku - Laravel Application) I see that heroku has changed its policies and requires credit card verification.

**Comment by danielgara** [comment]
Hi @developer-luis, the last time I researched Heroku free alternatives to deploy PHP applications, I found nothing. There are plenty of free alternatives for deploying JavaScript or Python apps, but I didn't find anything for PHP.

I will try to take another look in the next few weeks and I will let you know.
